vcr/revocation/statuslist2021_issuer.go

```diff
@@ -317,7 +318,9 @@ func (cs *StatusList2021) Entry(ctx context.Context, issuer did.DID, purpose Sta
 					return err
 				}
 
-				_, credRecord, err := cs.updateCredential(ctx, credentialIssuer, key)
+				// store transaction context
+				transactionContext := context.WithValue(ctx, storage.TransactionKey{}, tx)
+				_, credRecord, err := cs.updateCredential(transactionContext, credentialIssuer, kid)
```

Comment:
`cs.Credential()` and `ca.Entry()` also call `cs.updateCredential()` from within a DB transaction. Should these also wrap the context to prevent deadlocks? Not sure why they pass e2e test though...